pydantic/_internal/_fields.py

```diff
@@ -264,7 +264,10 @@ def collect_model_fields(  # noqa: C901
         # to make sure the decorators have already been built for this exact class
         decorators: DecoratorInfos = cls.__dict__['__pydantic_decorators__']
         if ann_name in decorators.computed_fields:
-            raise ValueError("you can't override a field with a computed field")
+            raise TypeError(
+                f'Field "{ann_name}" of class "{cls.__name__}" overrides symbol of same name in "{cls.__bases__[0].__name__}".\n'
```

Comment:
```suggestion
                f'Field {ann_name!r} of class {cls.__name__!r} overrides symbol of same name in {cls.__bases__[0].__name__!r}. '
```
a couple notes:
- The parent symbol might not be in `cls.__bases__[0]`, and I don't think there's a simple way to actually know which base class it is.
- Probably we should avoid explicit newlines in exception messages

Reply:
Good call